pom.xml

```diff
@@ -42,10 +42,10 @@
     <aws.version>1.11.272</aws.version>
     <aws.kcl.version>1.9.0</aws.kcl.version>
     <aws.kpl.version>0.12.8</aws.kpl.version>
-    <circus.train.version>11.4.0</circus.train.version>
+    <circus.train.version>12.0.0-SNAPSHOT</circus.train.version>
```

Comment:
Do we need a release for this?

Reply:
Not any more, the changes required are in the 12.0.0 release (via https://github.com/HotelsDotCom/circus-train/commit/533cfbbddba0925d894a1498e06a7a2b58c8f6c5) so we should just change this to 12.0.0. I'll do it now.